podcastparser.py

```diff
@@ -696,6 +732,8 @@ def parse_pubdate(text):
     'rss/channel/title': PodcastAttr('title', squash_whitespace),
     'rss/channel/link': PodcastAttrRelativeLink('link'),
     'rss/channel/description': PodcastAttr('description', squash_whitespace_not_nl),
+    'rss/channel/podcast:funding': PodcastAttrFromUrl('funding'),
```

Comment:
Should this be mapped to the existing `payment_url` attribute we already support?

If not, `funding_url` might be a better name.

Reply:
Changed to `funding_url`. Since the `payment_url` lives at the item level, but `funding_url` lives at the channel level, it's probably not wise to merge it.